### DISCUSSION TITLE
Looking for someone to take over the project

### DISCUSSION THREAD

**Discussion by WillowHayward** [arimgibson/Deadname-Remover#502] [Announcements]
Hi all,

Unfortunately despite my best intentions, I've been unable to work on this for a couple of years now. It's not really fair for that to be the thing that holds up progress, so I wanted to put out feelers for other people to take over the project? Specifically I'm looking for someone trustworthy (and ideally more available than myself) to transfer ownership of this repo as well as the Chrome and Firefox extension entries to. If you're interested could you drop a comment, or send me an email at WillHayCode@gmail.com

**Comment by arimgibson** [comment]
I would be very interested! Shooting you an email to talk further; I appreciate you realizing that you don't have the capacity but want this project to live on
